ROOTFS/etc/sage-utils/dns/tftp/config.txt

```diff
@@ -0,0 +1,23 @@
+#dtparam=i2s=on
+#dtparam=spi=on
+
+# Uncomment this to enable infrared communication.
+#dtoverlay=gpio-ir,gpio_pin=17
+#dtoverlay=gpio-ir-tx,gpio_pin=18
+
+# Additional overlays and parameters are documented /boot/overlays/README
+
+# Enable audio (loads snd_bcm2835)
+dtparam=audio=on
```

Comment:
we actually don't want any audio.  so should we comment this out?  please investigate.